packages/storybook-addon-baseline-grid/package.json

```
@@ -0,0 +1,106 @@
+{
+  "name": "@canonical/storybook-addon-baseline-grid",
+  "version": "0.0.0",
+  "description": "Displays the baseline grid",
+  "keywords": [
+    "storybook-addons", "storybook-addons"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "-"
+  },
+  "type": "module",
+  "license": "MIT",
```

```suggestion
  "license": "LGPL-3.0",
```

Updating license per [legal](https://sites.google.com/a/canonical.com/operations/legal/software-licensing)

---

Done